src/nanocurrency/util.py

```diff
@@ -3,6 +3,10 @@
 from ed25519_blake2b import VerifyingKey, SigningKey
 from bitarray import bitarray
 
+from .exceptions import CantReachServer
+
+import requests
```

Comment:
You need to add `requests` into `setup.py` as one of the dependencies to ensure it's installed.